SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security Policy
+
+## Supported Versions
+
+PMM versions starting from v2.0.0 are currently being supported.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.x.x   | :x:                |
```

Comment:
@rnovikovP who updates this page: https://www.percona.com/services/policies/percona-software-support-lifecycle 

This shows the correct EOL date for PMM but wrong color making it appear to still be supported.

Reply:
@shoffman-percona, I'll check the recent versions and ask IT to update the page.

Reply:
Update: this will take some time as some other products will be updated. ETA -  end of this week